src/Microsoft.AspNet.OData.Shared/Query/Expressions/SelectExpandBinder.cs

```diff
@@ -185,7 +185,8 @@ internal Expression CreatePropertyValueExpression(IEdmEntityType elementType, IE
             }
 
             string propertyName = EdmLibHelpers.GetClrPropertyName(property, _model);
-            Expression propertyValue = Expression.Property(source, propertyName);
+            var propertyInfo = source.Type.GetProperty(propertyName);
```

Comment:
Note that, as a general coding practice, we try not to use "var" in our production code.

Reply:
It is fixed. I found a lot of "var"s in codes:
![image](https://user-images.githubusercontent.com/50167116/65478911-e46f1a00-dea4-11e9-82f7-3e408ff2a766.png)

Is anyone fixing them?

Reply:
Thanks for the quick follow-up @komdil!

We haven't gone back through the code to remove var, but we are fixing them opportunistically as we update the code and are trying not to add any new uses of var in the product code.